cmd/server/main.go

```diff
@@ -282,6 +293,19 @@ func (o *ProxyRunOptions) Validate() error {
 		}
 	}
 
+	// validate the proxy strategies
+	if o.proxyStrategies != "" {
+		pss := strings.Split(o.proxyStrategies, ",")
+		for _, ps := range pss {
+			switch ps {
+			case string(server.ProxyStrategyDestHost):
+			case string(server.ProxyStrategyDefault):
+			default:
+				return fmt.Errorf("unknown proxy strategy: %s, available strategy is: destHost", ps)
```

Comment:
Again not blocking but would be nice if the error message listed all the valid strategy values.

Reply:
list all supported strategies in the error message